### PR TITLE
Add note about OS support

### PR DESCRIPTION
**Pull request by JAORMX**
To the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -826,3 +826,10 @@ $ oc extract cm/masters-scan-ip-10-0-174-253.ec2.internal-pod
 
 Note that if the results are too big for the ConfigMap, they'll be bzipped and
 base64 encoded.
+
+OS support
+==========
+
+Note that the current testing has been done in RHCOS. We're currently not
+testing in RHEL/CentOS nodes as one can simply run OpenSCAP directly on those
+OSes.
```
Comment (jhrozek) [inline README.md:835]: I know this is an upstream document, but is this going to be our recommendation for OCP, too?

Comment (JAORMX) [reply README.md:835]: I guess so... we haven't really tested in other distros yet. Would be nice! but haven't been able to :confused:

Comment (jhrozek) [reply README.md:835]: I guess what I tried to say is that the statement sounds definitive, like, we consider the current state OK. What do you think about saying "..in the absence of RHEL/CentOS support, one can simply(?) run OpenSCAP..."

**Comment by JAORMX** [comment]
@jhrozek rephrased it. What do you think?

**Comment by jhrozek** [comment]
Yet another issue of this heisenbug:
```
{"level":"info","ts":1597672078.5758018,"logger":"scanctrl","msg":"Node scan error","node.Name":"ip-10-0-189-23.us-west-2.compute.internal","errMsg":"the ConfigMap 'rhcos4-e8-worker-ip-10-0-189-23.us-west-2.compute.internal-pod' annotation was missing the result key"}
```
and as earlier the CM is annotated properly:
```
kind": "ConfigMap",
            "metadata": {
                "annotations": {
                    "compliance-remediations/processed": "",
                    "compliance.openshift.io/scan-error-msg": "",
                    "compliance.openshift.io/scan-result": "NON-COMPLIANT",
                    "openscap-scan-result/node": "ip-10-0-189-23.us-west-2.compute.internal"
                },
                "creationTimestamp": "2020-08-17T13:46:51Z",
                "labels": {
                    "compliance-scan": "rhcos4-e8-worker"
                },
                "managedFields": [
                    {
                        "apiVersion": "v1",
                        "fieldsType": "FieldsV1",
                        "fieldsV1": {
                            "f:data": {
                                ".": {},
                                "f:exit-code": {},
                                "f:results": {}
                            },
                            "f:metadata": {
                                "f:annotations": {
                                    ".": {},
                                    "f:compliance-remediations/processed": {},
                                    "f:compliance.openshift.io/scan-error-msg": {},
                                    "f:compliance.openshift.io/scan-result": {},
                                    "f:openscap-scan-result/node": {}
                                },
                                "f:labels": {
                                    ".": {},
                                    "f:compliance-scan": {}
                                }
                            }
                        },
                        "manager": "compliance-operator",
                        "operation": "Update",
                        "time": "2020-08-17T13:47:56Z"
                    }
                ],
                "name": "rhcos4-e8-worker-ip-10-0-189-23.us-west-2.compute.internal-pod",
```
I have created CMP-690 to track the issue.

**Pull request merged**